src/steps/OtherContractConsiderations/PII.vue

```diff
@@ -117,5 +116,17 @@ export default class PII extends Vue {
       value: "No",
     },
   ];
+
+  public get selectedPIIOption(): string {
+    const included = PIIRecord.PIIRecordIncluded;
+    if (included !== null) {
+      return included ? "Yes" : "No";
+    }
+    return "";
+  }
+
+  public set selectedPIIOption(value: string) {
+    PIIRecord.setPIIRecord(value === "Yes" ? true : false);
```

Comment:
I had it that way originally... shorter is `(value === "Yes")` and forget the true/false - it's unneeded